documentation/modules/managing/con-custom-resources-status.adoc

```diff
@@ -56,6 +60,10 @@ m¦KafkaRebalance
 ¦`KafkaRebalance` schema reference
 ¦The status and results of a rebalance
 
+m¦StrimziPodSet
+¦`StrimziPodSetStatus` schema reference
```

Comment:
This table seems a bit weird. For most items (Apart from the `KafkaRebalance`), you refer to them as `KafkaStatus`, KafkaNodePoolStatus`, etc. But this is a name that has no meaning outside of the API reference. I would expect to not refer to them (and talk only about e.g. `Kafka resource status section` ) or include a link to the API reference to make it more clear what does it mean.

Reply:
I'm not clear on this comment. The schema column lists the schemas that contain status properties and we have a link to the Schema reference guide above the table.  That seems okay to me. Could you expand on what else you're expecting?